packages/microcosm-www-next/src/pages/index.js

```diff
@@ -1,40 +1,152 @@
 import React from 'react'
+import data from '../data/index.json'
+import { Graphic1, Graphic2, Graphic3 } from '../components'
 
-const IndexPage = () => (
-  <section className="section">
-    <div className="wrapper">
-      <h2 className="section-title">
-        <span>01.</span>
-        Rendering a View
-      </h2>
-
-      <div className="figure-container">
-        <figure className="figure -left">
-          <img src="" className="figure__graphic" alt="TODO" />
-          <figcaption className="figure__content">
-            <h3 className="figure__content__header">In the browser</h3>
-            <p className="figure__content__text">
-              A user fires up the browser to take a quiz.
-            </p>
-          </figcaption>
-        </figure>
-
-        <figure className="figure -right">
-          <img src="" className="figure__graphic" alt="TODO" />
-          <figcaption className="figure__content">
-            <h3 className="figure__content__header">In Microcosm</h3>
-            <p className="figure__content__text">
+export default class IndexPage extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+
+  componentDidMount() {
+    // TODO later - accessibility to alert on change
+    this.setVars()
+    this.beginObserve()
+  }
+
+  setVars() {
+    this.onMicrocosmView = true
+    this.currentSection = 1
+    this.observeOptions = {
+      root: null,
+      rootMargin: '0px 0px 100px',
+      threshold: 1.0,
+    }
+
+    this.text = document.getElementById('text')
+    this.heading = document.getElementById('heading')
+    this.subheadingTop = document.getElementById('subheading-top')
+    this.subheadingBottom = document.getElementById('subheading-bottom')
+
+    let graphics = document.querySelectorAll('[data-module="ObserveGraphic"]')
+    this.graphicsMap = this.setGraphicsMap(graphics)
+  }
+
+  setGraphicsMap(graphics) {
+    return [].slice.call(graphics).reduce((map, graphic) => {
+      map[graphic.dataset.section] = graphic
+      return map
+    }, {})
+  }
```

Comment:
Could you derive this information from the index.json file instead of reading from the DOM?